### PR TITLE
Add GALA app as an example of user application

### PR DESCRIPTION
**Pull request by unbel13ver**
As GALA app is a closed source peace of software yet, let's have it as pre-built for now.

I'd like to discuss directory structure for adding user applications - I've added new directory into the srcroot for now.
I also do not like the idea of having monolithic `weston.ini` file, but I did not get how to append text from other files/nixos modules.
Ideally, I'd like to add, for instance, weston launcher from the package's .nix file.

And yes, currently there is a mock icon used for Gala launcher.

### CODE REVIEW

**Comment by vilvo** [comment]
Got confirmation that there's an EC decision from MFA Finland (14.12.2020 UHA2020-007419 "EC permit not needed") for closed source delivery of GALA. In future, please check future PRs with your PMs for permit before opening a PR with closed binaries. No exceptions. It's a legal compliance requirement. Default is open source delivery for EC compliance.

File: user-apps/gala/default.nix [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        --set-interpreter "${dynamic-linker}" \
+        --set-rpath "${rpath}" \
+        ${targetPath}/dev.scpp.saca.gala
+
+      mkdir -p $out/bin
+      ln -s $out/gala/dev.scpp.saca.gala $out/bin/gala
+    '';
+
+    meta = with lib; {
+      description = "Google Andoid look-alike";
```
Comment (vilvo) [inline user-apps/gala/default.nix:132]: there's a typo - should be *Android*

**Review by vilvo — APPROVED** [review]
lgtm

**Comment by unbel13ver** [comment]
> Got confirmation that there's an EC decision from MFA Finland (14.12.2020 UHA2020-007419 "EC permit not needed") for closed source delivery of GALA. In future, please check future PRs with your PMs for permit before opening a PR with closed binaries. No exceptions. It's a legal compliance requirement. Default is open source delivery for EC compliance.

Thanks for double checking this! I checked with PM before doing the work, I should have mentioned that in the comment.

File: user-apps/gala/default.nix (hunk 1 above)
Comment (unbel13ver) [reply user-apps/gala/default.nix]: Addressed, thanks for noticing this.

Comment (vilvo) [reply user-apps/gala/default.nix]: Sure, implies I've actually read the PR 😄 - thanks for addressing a minor typo.

**Comment by vilvo** [comment]
> > Got confirmation that there's an EC decision from MFA Finland (14.12.2020 UHA2020-007419 "EC permit not needed") for closed source delivery of GALA. In future, please check future PRs with your PMs for permit before opening a PR with closed binaries. No exceptions. It's a legal compliance requirement. Default is open source delivery for EC compliance.
> 
> Thanks for double checking this! I checked with PM before doing the work, I should have mentioned that in the comment.

Sure. I have not been closely involved with GALA dev so I had to check as us (or me reviewing/approving) integrating closed source SW to ghaf is export. Liabilities are individual, not corporate.

**Review by nesteroff — APPROVED** [review]

**Comment by brianmcgillion** [comment]
After some fiery hoops. Verified. Only comment is the icon looks like the terminal. But loads otherwise.

**Pull request merged**